paasta_tools/frameworks/native_service_config.py

```diff
@@ -243,7 +243,7 @@ def base_task(self, system_paasta_config: SystemPaastaConfig, portMappings=True)
                 ],
             },
             'command': {
-                'value': self.get_cmd(),
+                'value': str(self.get_cmd()),
```

Comment:
Can't this be:

```
'value': self.format_cmd()
```

And if the native thing needs to always be a string, then it's format_cmd method can do its own `str`?